mongodb/src/test/java/org/hibernate/ogm/datastore/mongodb/test/options/writeconcern/WriteConcernPropagationTest.java

```diff
@@ -114,7 +117,9 @@ public void shouldApplyConfiguredWriteConcernForRemoveTuple() {
 		session.close();
 
 		// then expect a call to remove with the configured write concern
-		verify( mockClient.getCollection( "GolfPlayer" ) ).remove( any( DBObject.class ), eq( WriteConcern.MAJORITY ) );
+		// verify( mockClient.getCollection( "GolfPlayer" ) ).remove( any( Document.class ), eq( WriteConcern.MAJORITY )
```

Comment:
We typically don't comment out code, unless there's a very specific reason which might require an explanation comment